staker-rewards/rewards-merkle-generator.sh

```diff
@@ -7,24 +7,91 @@ LOG_END='\n\e[0m' # new line + reset color
 
 WORKDIR=$PWD
 
+# default file for calculated staker reward allocation
+STAKER_REWARD="distributor/staker-reward-allocation.json"
+
+NODE_CURRENT_VER="$(node --version)"
+NODE_REQUIRED_VER="v14.3.0"
+
+if [ "$(printf '%s\n' "$NODE_REQUIRED_VER" "$NODE_CURRENT_VER" | sort -V | head -n1)" != "$NODE_REQUIRED_VER" ]; 
+then
+      echo "Required node version must be at least ${NODE_REQUIRED_VER}" 
+      exit 1
+fi
+
+help()
+{
+   echo ""
+   echo "Usage: $0 -v -h <eth_host> -s <start_interval> -e <end_interval> -r <rewards> -u <tenderly_url> -t <tenderly_token>"
+   echo -e "\t-v Optional verify flag just to display the results without generation of a merkle tree"
+   echo -e "\t-h Websocket endpoint of the Ethereum node"
+   echo -e "\t-s Start of the interval passed as UNIX timestamp"
+   echo -e "\t-e End of the interval passed as UNIX timestamp"
+   echo -e "\t-r Total KEEP rewards distributed within the given interval passed as 18-decimals number"
+   echo -e "\t-u Optional Tenderly API project URL"
+   echo -e "\t-t Optional access token for Tenderly API used to fetch transactions from the chain"
+   exit 1 # Exit script after printing help
+}
+
+if [ "$1" == "-help" ]; then
+  help
+fi
+
+printf "${LOG_START}Processing input parameters...${LOG_END}"
+
+while getopts "vh:s:e:r:u:t:" opt
+do
+   case "$opt" in
+      v ) verify=true ;;
+      h ) eth_host="$OPTARG" ;;
```

Comment:
Thoughts on renaming this option to `w` (websocket)? `h` is usually reserved for help.

Reply:
Sure, done 37c7db5